packages/foam-vscode/src/features/open-daily-note.ts

```diff
@@ -83,7 +85,10 @@ async function createDailyNoteDirectoryIfNotExists(dailyNotePath: string) {
 
 async function focusDailyNote(dailyNotePath: string) {
   const document = await workspace.openTextDocument(Uri.parse(dailyNotePath));
-  window.showTextDocument(document);
+  const editor = await window.showTextDocument(document);
+  // Move the cursor to line 2 (0 indexed)
+  const { range } = editor.document.lineAt(2);
```

Comment:
is the goal to always move to the end of the daily note? or to land the cursor on the line following the title?

Reply:
Ah! Thanks! Yes, that is an oversight by me. Will fix it.

Reply:
Done!